### PR TITLE
finance tab constant

### LINKED ISSUE #69: Finance Tab Constant
Module: Finance

Usage
- A constant for Finance Module Tab

- [x] Development

### PR DESCRIPTION
**Pull request by CarlosSia**
Resolve #69 

UI Screenshot:
### Event Listing Tab:
**Before:**
Screenshot_708

**After:**
Screenshot_713

### Event Details Tab:
**Before:**
Screenshot_709
Screenshot_710

**After:**
Screenshot_714
Screenshot_715

### Finance Tab:
**Before:**
Screenshot_711

**After:**
Screenshot_712

### CODE REVIEW

File: src/main/webapp/app/shared/util/tab.constants.ts [hunk 1]
```diff
@@ -45,3 +45,26 @@ export const eventTabList = (eventId: string): ITabInfo[] => [
     tabRoute: `/entity/event-crew/event/${eventId}`
   }
 ];
+
+export const financeTabList: ITabInfo[] = [
+  {
+    tabName: 'Report',
+    tabTranslateKey: 'clubmanagementApp.event.upcoming',
```
Comment (HowardYaw) [inline src/main/webapp/app/shared/util/tab.constants.ts]: what is upcoming?

**Comment by HowardYaw** [comment]
@CarlosSia Can include the change of css in th PR?
and provide Screenshot for latest tab UI
- Event Listing Tab
- Event Details Tab
- Finance Tab

File: src/main/webapp/app/shared/util/tab.constants.ts (hunk 1 above)
Comment (CarlosSia) [reply src/main/webapp/app/shared/util/tab.constants.ts]: Corrected it in https://github.com/Five-Fishes/Club-Management-React-Client/pull/78/commits/8a0dac9be11658f44681f25535fbbbd24578f440

**Comment by CarlosSia** [comment]
> @CarlosSia Can include the change of css in th PR?
> and provide Screenshot for latest tab UI
> 
> * Event Listing Tab
> * Event Details Tab
> * Finance Tab

I had add all screenshots in first comment

**Review by HowardYaw — APPROVED** [review]

**Review by kachun333 — APPROVED** [review]

**Pull request merged**